src/app/containers/Preferences/index.styl

```diff
@@ -477,4 +488,60 @@
 .disabled-message {
   color: #b82e2e;
   font-weight: bold;
+}
+
+.color-button {
+  padding: 9px;
+  border-radius: 50%;
+  border-width: 0px;
+}
+
+.label {
+  font-size: 1.35rem
+}
+
+.color-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 5px;
+}
+
+.color-display {
+  height: 25px;
+  width: 140px;
+  min-width: 140px;
+  background-color: #B2B2BD
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: .25rem;
+  padding-left: 3px;
+  padding-right: 3px;
+
+  margin-left: 0.5em;
+}
+
+.dots {
+  overflow: hidden;
+}
+
+.dots:after {
+  content: '. . . . . . . . . . . . . . . . . . . . . . . . . . . . .';
+  color: #000;
+  white-space: nowrap;
+}
```

Comment:
Think this is causing a UI bug where the container of the preferences is breaking out of the modal container. There are two easier ways to handle this, could not have these dots and just add more spacing between each color picker section, or you could replace this with a line div where you create a div and set its background and set its height to something small so it looks like a line.